### ISSUE TITLE
Please Add  http://suttonhoney.com - verified

### ISSUE THREAD

**Issue by CreativeName4** [acceptbitcoincash/acceptbitcoincash#388]
**Submission Notes:** Please fill in the template below, and don't worry about exact formatting, just do your best! Use the _**Preview**_ tab to check your submission before sending it. Help for **region/country 2-letter codes** are below.

---------------
Requesting to add {{**Sutton Honey**}} to the {{**Retail**}} category. Details follow:

```yml

- name: Sutton Honey
      url: http://suttonhoney.com
      img: http://suttonhoney.com/pics/sunfieldbanner.jpg
      twitter: (none)
      facebook: (none)
      region: NA
      country: CA
      city: Sutton
      bch: yes
      btc: yes
      othercrypto: yes
      doc: http://suttonhoney.com/
```

Add more information here if necessary.

For 2-letter _region_ codes, use:

Code | Name
-- | --
af | Africa
na | North America
oc | Oceania
an | Antarctica
as | Asia
eu | Europe
sa | South America

For 2-letter _country_ codes, use [ISO_3166-1_alpha-2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Officially_assigned_code_elements) codes.

**Comment by kenman345** [comment]
Would love a square image for this

**Comment by kenman345** [comment]
fixed in PR #417
